README.md

```diff
@@ -811,10 +811,10 @@ Other Style Guides
     console.log(...x);
 
     // bad
-    new (Function.prototype.bind.apply(Date, [null, 2016, 8, 5]));
+    const date = new (Function.prototype.bind.apply(Date, [null, 2016, 8, 5]));
```

Comment:
This doesn't add any value (if you're applying a linter rule, it'd error on the duplicate const definition anyways). Please excise this commit.